src/components/Pages/Location/LocationInfo.js

```diff
@@ -128,11 +120,13 @@ const LocationPageFacilityHours = ({hours}) => {
           {getDaysInOrder().map((day, i) => (
             <tr key={i}>
               <td>
-                {intl.formatMessage(
-                  i18nDate['weekday' + capitalize(day)],
-                )}
+                {intl.formatMessage(i18nDate['weekday' + capitalize(day)])}
+              </td>
+              <td>
+                {hours[day] !== null
+                  ? hours[day]
+                  : intl.formatMessage(i18nLocations.closed)}
```

Comment:
instead of setting hours to "Closed" in cleanData, we just set it to null and localize here